README.md

```diff
@@ -43,6 +43,22 @@ This boilerplate comes with batteries included, you’ll find:
 4. Create a new Git repository (`git init`)
 5. Create the initial Git commit (`git commit -a -m "Initial commit"`)
 
+## Preferred libraries
+
+Some batteries aren’t included since all projects have their own needs and requirements. Here’s a list of our preferred libraries to help you get started:
+
+| Category                    | Libraries                                                                                           |
+| --------------------------- | --------------------------------------------------------------------------------------------------- |
+| Authentication              | [`ueberauth`](https://github.com/ueberauth/ueberauth), [`pow`](https://github.com/danschultzer/pow) |
+| Asynchronous job processing | [`oban`](https://github.com/sorentwo/oban)                                                          |
+| Emails                      | [`bamboo`](https://github.com/thoughtbot/bamboo), [`swoosh`](https://github.com/swoosh/swoosh)      |
+| File upload                 | [`waffle`](https://github.com/elixir-waffle/waffle)                                                 |
+| HTTP client                 | [`tesla`](https://github.com/teamon/tesla)                                                          |
```

Comment:
I believe we enforced Hackney as the default HTTP client in [mix.exs](https://github.com/mirego/elixir-boilerplate/blob/master/mix.exs#L45) due to the latest New Relic dependencies. Not sure if it should removed or if we should change the information here?

Reply:
`hackney` is required by a lot of libraries, but I still think `tesla` should be recommend for a more high level usage.

Reply:
Yep, I agree with @garno.

I think Tesla is a much more developer-friendly HTTP client when building business logic code.

I think Hackney is considered more like a low-level HTTP client.

Reply:
👍